test/fixtures/shared/terraform.tfvars

```diff
@@ -0,0 +1,8 @@
+FIXME="FIXME REMOVE ID AND DIR"
```

Comment:
We should only commit `terraform.tfvars.sample`.

Reply:
The follow on to this comment being `terraform.tfvars` should probably land in `.gitignore`.

Reply:
Fixed